packages/openactive-integration-tests/test/helpers/order-utils.js

```diff
@@ -82,9 +84,30 @@ const getOrderId = pipe(
   assertValueSatisfiesPredicateIfExists(isString, 'orderId (@id) is not a string'),
 );
 
+/**
+ * @param {{
+ * orderedItem: OrderItem[]
+ * }} order
+ */
+function createPositionOrderIntakeFormMap(order) {
+  /** @type {Object.<number, OrderItemIntakeForm>} */
+  let map;
+
+  for (const orderItem of order.orderedItem) {
+    if (!isNil(orderItem.orderItemIntakeForm)) {
+      map = {
+        ...map,
+        [orderItem.position]: orderItem.orderItemIntakeForm,
+      };
```

Comment:
I mean I can store them as strings because when I get them out again, they come out as strings (I use Object.keys which returns an array of strings). Probably easier all round to have them as strings, will change!